src/fights.ts

```diff
@@ -707,6 +708,19 @@ function molemanReady() {
   return have($item`molehill mountain`) && !get("_molehillMountainUsed");
 }
 
+const stunDurations = new Map<Skill | Item, Delayed<number>>([
+  [$skill`Blood Bubble`, 1],
+  [
+    $skill`Entangling Noodles`,
+    () => (myClass() === $class`Pastamancer` && !have($skill`Shadow Noodles`) ? 1 : 0),
+  ],
+  [$skill`Frost Bite`, 1],
+  [$skill`Shadow Noodles`, 2],
+  [$skill`Soul Bubble`, () => (mySoulsauce() >= 5 ? 2 : 0)],
+  [$skill`Summon Love Gnats`, 1],
+  [$item`Rain-Doh blue balls`, 1],
```

Comment:
I would also conditionally add

- Club Foot: requires SC, 2+ Fury & wielding a club to Stun. Otherwise just Staggers
- Shell Up: requires TT and Blessing of the Storm Tortoise to Stun. Otherwise just Staggers
- Accordion Bash: requires AT & wielding an accordion to Stun. Otherwise just Staggers

Lost Comb is also a re-usable 3 round Stun combat item. Trivial to acquire for anyone with the VIP room key.

Reply:
Sauceshell too, which has no reqs and does some chip damage

Reply:
The Lost Comb is used up after 3 uses just btw.

Reply:
> Sauceshell too, which has no reqs and does some chip damage

I think the intention of this is to kill specifically with Robo Kiss, therefore any extra chip damage is superfluous and doesn't actually help us.

If the user doesn't have enough stuns, we should use the normal glitch kill anyway.

Reply:
I'll add shell up, the rest are too conditional.